pillar.example

```diff
@@ -10,3 +10,4 @@ docker-pkg:
     version: 1.6.2
     refresh_repo: false
     process_signature: /usr/bin/docker
+    pip_version: '<= 1.2.3'
```

Comment:
Comment this line and add a comment what this is good for. Too many people will just copy and paste pillar.example files and end up with weird versions.